package.json

```diff
@@ -1,13 +1,14 @@
 {
   "dependencies": {
-    "@scalar-labs/scalardl-javascript-sdk-base": "^3.0.2",
-    "dexie": "^3.0.2",
-    "google-protobuf": "^3.13.0",
+    "@scalar-labs/scalardl-javascript-sdk-base": "3.1.0",
+    "dexie": "^3.0.3",
+    "electron-to-chromium": "^1.3.690",
```

Comment:
This package is required when building this web-sdk as a `dist` package.

Reply:
Just to make sure, this package is not only required when building but also at runtime when using the sdk as a `dist` package?

Reply:
I saw it was required by these stacks
```
Require stack:
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/browserslist/index.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/config/browserslistTargetHandler.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/config/target.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/config/defaults.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/webpack.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/index.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack-cli/lib/webpack-cli.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack-cli/lib/bootstrap.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack-cli/bin/cli.js
- /Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:15)
    at Function.Module._load (node:internal/modules/cjs/loader:763:27)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/browserslist/index.js:5:11)
    at Module._compile (/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
    at Module.load (node:internal/modules/cjs/loader:967:32)
    at Function.Module._load (node:internal/modules/cjs/loader:807:14)
    at Module.require (node:internal/modules/cjs/loader:991:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/browserslist/index.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/config/browserslistTargetHandler.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/config/target.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/config/defaults.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/webpack.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/lib/index.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack-cli/lib/webpack-cli.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack-cli/lib/bootstrap.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack-cli/bin/cli.js',
    '/Users/sianghwa/go/src/github.com/scalarindetail/scalardl-web-client-sdk/node_modules/webpack/bin/webpack.js'
  ]
}
```

Reply:
I pulled the branch locally and tried to reproduce the problem without success. Can you describe the step to reproduce it ? 

On my side, I did the following : 

0. Remove `"electron-to-chromium": "^1.3.690", from packages.json. 
1. Remove `node_modules` folder to avoid using some lib that differs from package-lock and ran `npm install`
2. Ran `npm test` successfully 
3. Generated static library with `make dist` and ran some successfully manual test on `example/index.html` using Firefox.

Reply:
@Torch3333 hmm, i can't reproduce the error anymore, even after i uninstall `electron-to-chromium` globally

Reply:
Ok, I guess it's safe to assume it was a mishap and we can remove it from package.json. 

I checked and this package is still added as a dev dependency to package-lock.json so it is required eventually for one of the dependency anyway.

Let me push an updated package-lock.